.github/ISSUE_TEMPLATE.md

```diff
@@ -1,11 +1,13 @@
-<!-- This form is for bug reports and feature requests ONLY! 
+<!-- This form is for bug reports and feature requests ONLY!
 
 If you're looking for help check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
+
+If this may be security issue, please disclose it privately via https://kubernetes.io/security/.
```

Comment:
"may be a security issue"

Reply:
Other URLs use markdown… not sure if it matters or even renders properly

Reply:
Fixed typo.

Yeah, I don't know why they use markdown... If this is rendered as markdown it won't show up at all, since it's in a comment block.